nibabel/__init__.py

```diff
@@ -7,6 +7,8 @@
 #
 ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
 
+# ruff: noqa: F401
```

Comment:
We should set file-level ignores for `__init__.py`, rather than encoding this in the file.

Reply:
Done